browser/net/brave_network_delegate_browsertest.cc

```diff
@@ -61,9 +61,19 @@ class BraveNetworkDelegateBrowserTest : public InProcessBrowserTest {
 
     third_party_cookie_url_ =
         embedded_test_server()->GetURL("b.com", "/set-cookie?name=Good");
+    first_party_cookie_url_ =
+        embedded_test_server()->GetURL("a.com",
+                                       "/set-cookie?name=Good");
     subdomain_first_party_cookie_url_ =
         embedded_test_server()->GetURL("subdomain.a.com",
                                        "/set-cookie?name=Good");
+
+    domain_registry_url_ = embedded_test_server()->GetURL("mobile.twitter.com",
+                                                        "/cookie_iframe.html");
+    iframe_domain_registry_url_ =
+        embedded_test_server()->GetURL("blah.twitter.com",
```

Comment:
for some reason just using "twitter.com" would fail to load in the test server, but this serves the same purpose